### PR TITLE
CR Transition admin drafts

### PR DESCRIPTION
**Pull request by iherman**
Prepared the texts for

1. preliminary publication request
2. official publication request
3. AC mailing draft
4. Home Page news drafts

All, except (1), will be sent out if and when the transition is approved. (1) will be sent out as soon as the WG has officially approved the request for transition.

A check for spelling mistakes, copy/paste errors, etc, is welcome.

### CODE REVIEW

File: admin/CR-March-2021/CRACAnnouncement.txt [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+[DRAFT] Decentralized Identifiers (DIDs) v1.0 is a W3C Candidate Recommendations (Call for Implementations)
+-----------------------------------------------------------------------------------------------------------
+
+
+Dear Advisory Committee Representative,
+Chairs,
+
+I am pleased to announce that the following documents is now a Candidate Recommendations:
```
Comment (msporny) [inline admin/CR-March-2021/CRACAnnouncement.txt]:
```suggestion
I am pleased to announce that the following document is now a Candidate Recommendation:
```

File: admin/CR-March-2021/CRHPN.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+To: xueyuan, Cc: coralie, comm team, denis, jennyliang@w3.org, DID WG Chairs, Manu
+
+
+Subject: [DRAFT HPN] Candidate Recommendations published for Decentralized Identifiers (DIDs) v1.0.
```
Comment (msporny) [inline admin/CR-March-2021/CRHPN.txt]:
```suggestion
Subject: [DRAFT HPN] Candidate Recommendation published for Decentralized Identifiers (DIDs) v1.0.
```

**Review by msporny — APPROVED** [review]
Minor editorial nits, otherwise, LGTM.

File: admin/CR-March-2021/CRACAnnouncement.txt [hunk 3]
```diff
@@ -0,0 +1,69 @@
+To: xueyuan, Cc: coralie, comm team, denis, jennyliang@w3.org, DID WG Chairs, Manu
+
+
+[DRAFT] Decentralized Identifiers (DIDs) v1.0 is a W3C Candidate Recommendations (Call for Implementations)
```
Comment (brentzundel) [inline admin/CR-March-2021/CRACAnnouncement.txt]: Not sure if this plural is a typo or deliberate:
```suggestion
[DRAFT] Decentralized Identifiers (DIDs) v1.0 is a W3C Candidate Recommendation (Call for Implementations)
```

File: admin/CR-March-2021/CRACAnnouncement.txt [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+-----------------------------------------------------------------------------------------------------------
+
+
+Dear Advisory Committee Representative,
```
Comment (brentzundel) [inline admin/CR-March-2021/CRACAnnouncement.txt]:
```suggestion
Dear Advisory Committee Representatives,
```

File: admin/CR-March-2021/preliminary_publ_request.txt [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+I have already put the document on TR space:
+
+https://www.w3.org/TR/2021/CR-did-core-20210318/
+
+and the lead editors of the document (Manu Sporny and Amy Guy, cc-d) have gone through the pubrules' checker. However, 
+we would appreciate if you could check the document whether it is ready for publication, or whether
+we have to make some editorial changes.
+
+Of course, as soon as we get the transition approval, I will send you (ie, the webmaster) and official
```
Comment (brentzundel) [inline admin/CR-March-2021/preliminary_publ_request.txt]:
```suggestion
Of course, as soon as we get the transition approval, I will send you (ie, the webmaster) an official
```

**Review by brentzundel — APPROVED** [review]
Found a couple small typos, otherwise looks great.

File: admin/CR-March-2021/preliminary_publ_request.txt (hunk 5 above)
Comment (TallTed) [inline admin/CR-March-2021/preliminary_publ_request.txt]:
```suggestion
Of course, as soon as we get the transition approval, I will send you (i.e., the webmaster) an official
```

**Comment by TallTed** [comment]
Some filenames ending in `.txt` contain HTML, and line widths vary wildly.

So, are these txt or html?  Are they to be dispatched by email, or through website publication, or both?  

I would suggest different adjustments, depending on those answers.

**Comment by iherman** [comment]
Ted, it is an email I will send to our comm team who then will use it to put up an HTML on the Website. Ie, the line breaks do not really matter, but better not to get them too long because email clients may not like it.

File: admin/CR-March-2021/preliminary_publ_request.txt [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+I have already put the document on TR space:
+
+https://www.w3.org/TR/2021/CR-did-core-20210318/
+
+and the lead editors of the document (Manu Sporny and Amy Guy, cc-d) have gone through the pubrules' checker. However, 
+we would appreciate if you could check the document whether it is ready for publication, or whether
+we have to make some editorial changes.
+
+Of course, as soon as we get the transition approval, I will send you (ie, the webmaster) an official
```
Comment (TallTed) [inline admin/CR-March-2021/preliminary_publ_request.txt]:
```suggestion
Of course, as soon as we get the transition approval, I will send you (i.e., the webmaster) an official
```

**Pull request merged**